ethanschreur/kids-krafts
# Customers can pay for their order with Stripe

**What does this PR do?**
- This pull request allows customers to pay for their order through Stripe and view a success page.

**Description of the task to be completed** 
- Add a checkout process using the Stripe API
- Configure an email receipt to be sent to customers after they made a purchase
- Create a "Success Page" that customers are redirected to after making a purchase
- The "Success Page" should have details about how to pick up their order.

**How should this  be tested**
- (On Windows) Run `. venv/scripts/activate` 
- `python run.py`
- Go to http://127.0.0.1:5000/shop
- Add products to your cart
- Then go to http://127.0.0.1:5000/order_details
- Select a pickup date and time
- Press the pay button
- Confirm that it takes you to Stripe
- Enter in your information and use 4242 4242 4242 4242 for your credit card information
- Press Pay
- Check that the browser redirects you to the success page
- Check that it displays contains your email
